MIT.md

```diff
@@ -1,8 +1,5 @@
 ## Copyright 2021, smunoz1988
 
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
-
 Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
 
 The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
```

Comment:
- **[OPTIONAL]** I am sorry that it wasn't pointed out earlier, I highly recommend changing the word "[APP TYPE]" to either "website" or "web app" as you desire 👍.